frontend/pages/for-investors.tsx

```diff
@@ -0,0 +1,451 @@
+                      whatCanDoIndex === 2,
+                  }
+                )}
+              >
+                {whatHecoCanDoImages.map((imageSrc, index) => {
+                  return (
+                    <div
+                      key={imageSrc + index}
+                      className={cx(
+                        'overflow-hidden rounded-2xl transition-all ease-in-out duration-700',
+                        {
+                          'w-[calc(100vw-20px)] lg:w-[50vw] opacity-100': index === whatCanDoIndex,
+                          'w-[calc(100vw-20px)] lg:w-[33.33vw]': index !== whatCanDoIndex,
+                          'opacity-40': index > whatCanDoIndex,
+                          'opacity-0': index < whatCanDoIndex,
+                        }
+                      )}
+                    >
+                      <Image
+                        src={imageSrc}
+                        alt=""
+                        width={isXl ? 1063.5 : 709}
+                        height={isXl ? 750 : 500}
+                        objectFit="cover"
+                        objectPosition="center"
+                        layout="fixed"
+                        className="rounded-2xl"
+                      />
+                    </div>
+                  );
+                })}
+              </div>
+              <div className="lg:absolute p-7 -translate-y-8 lg:-translate-y-full max-w-full lg:max-w-[500px] lg:left-1/2 bg-background-light rounded-xl -mt-1.5">
+                <div className="flex justify-between mb-2 align-middle">
+                  <div>
+                    <p className="text-xl font-bold text-gray-600">
+                      <span className="text-green-dark">0{whatCanDoIndex + 1}</span> / 03
+                    </p>
+                  </div>
+                  <div className="flex gap-2.5">
+                    <button
+                      className="flex items-center justify-center w-8 h-8 px-0 py-0 border rounded-full border-beige drop-shadow-lg "
+                      onClick={() =>
+                        setWhatCanDoIndex(whatCanDoIndex === 0 ? 2 : whatCanDoIndex - 1)
+                      }
+                    >
+                      <Icon
+                        icon={ChevronLeft}
+                        className="text-black transition-colors duration-300 ease-in-out hover:text-beige"
+                      />
+                    </button>
+                    <button
+                      className="flex items-center justify-center w-8 h-8 p-0 border rounded-full drop-shadow-lg "
+                      onClick={() =>
+                        setWhatCanDoIndex(whatCanDoIndex === 2 ? 0 : whatCanDoIndex + 1)
+                      }
+                    >
+                      <Icon
+                        icon={ChevronRight}
+                        className="text-black transition-colors duration-300 ease-in-out hover:text-beige"
+                      />
+                    </button>
```

Comment:
Please reuse the `<Button />` component to get the hover, focus, etc. styles and  add some labels (you can use a `<span />` with a `sr-only` class).